var/spack/repos/builtin/packages/py-nose-cov/package.py

```diff
@@ -9,7 +9,7 @@
 class PyNoseCov(PythonPackage):
     """This plugin produces coverage reports."""
 
-    homepage = "http://bitbucket.org/memedough/nose-cov/overview"
+    homepage = "https://pypi.org/project/nose-cov/"
```

Comment:
You can remove this line, it's the default set by `pypi`

Reply:
This should be fixed.